CMakeLists.txt

```diff
@@ -217,11 +217,72 @@ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/zenoh-pico
   COMPONENT Headers
 )
 
-if(BUILD_EXAMPLES)
-  add_subdirectory(examples)
```

Comment:
Separate "examples/CMakeLists.txt" allowed to check if the zenoh-pico's root CMakeLists.txt works good when it's included as a subproject. What's the necessity to remove this feature?

Reply:
I am doubling checking, and this one was not removed. It is still in the latest commit.